File: Atcoder_#351/C/a_the_bottom_of_the_ninth.c
#include <stdio.h>

int main() {
    int A[9], B[8];
    int sum_A = 0, sum_B = 0;

    // チーム高橋の得点を入力
    for (int i = 0; i < 9; i++) {
        scanf("%d", &A[i]);
        sum_A += A[i];
    }

    // チーム青木の得点を入力
    for (int i = 0; i < 8; i++) {
        scanf("%d", &B[i]);
        sum_B += B[i];
    }

    // 勝利するために必要な最小得点を計算
    int required_score = (sum_A - sum_B) + 1;

    // 結果を出力
    printf("%d\n", required_score);

    return 0;
}

File: Atcoder_#351/javascript/a_the_bottom_of_the_ninth.js
const readline = require('readline');

// 標準入力を読み取るためのインターフェースを作成
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

let inputLines = [];

// 標準入力を読み取るイベントリスナー
rl.on('line', (line) => {
  inputLines.push(line);
}).on('close', () => {
  // 入力データを処理する関数を呼び出し
  processInput(inputLines);
});

function processInput(input) {
  // 各行の文字列を配列に分割し、各文字を数値に変換
  const a = input[0].trim().split(' ').map(Number);
  const b = input[1].trim().split(' ').map(Number);

  // 配列の合計を計算
  const sumA = a.reduce((acc, val) => acc + val, 0);
  const sumB = b.reduce((acc, val) => acc + val, 0);

  // 差を計算
  const score_difference = sumA - sumB;

  // 結果を出力
  console.log(score_difference + 1);
}
